test/translator/tc_iso19115_2_to_dcatus.rb

```diff
@@ -93,4 +92,16 @@ def test_theme
       res = dcatusNS.build(@@intMetadata)
       assert_equal('biota farming', res)
    end
+
+   def test_identifier
+      expected = 'ISO19115-2-ID-123456'
+
+      assert_equal(expected, @@intMetadata[:metadata][:fileIdentifier])
+   end
+
+   def test_is_part_of
+      expected = 'ISO19115-2-ID-123456-parent'
+
+      assert_equal(expected, @@intMetadata[:metadata][:parentIdentifier])
+   end
```

Comment:
the pattern for these unit tests are to `build` from the associated dcatus writer module instead of simply pulling directly from the internal metadata object (i.e. `@@intMetadata` ). change these to do what every other test is doing.

Reply:
so for identifier 
```python 
dcatusNS = ADIWG::Mdtranslator::Writers::Dcat_us::Identifier
res = dcatusNS.build(@@intMetadata)
# make assertions here....
```